tools/crm_resource.c

```diff
@@ -487,23 +488,22 @@ main(int argc, char **argv)
             case 'I':
                 interval = optarg;
                 break;
-
+            
+            case 'D':
+                require_crmd = TRUE;    
+                require_dataset = FALSE; 
```

Comment:
Since 'D' will fall through to 'F', it will still get require_crmd. You'll probably need to break 'D' out into its own block.